themes/default/content/docs/get-started/kubernetes/review-project.md

```diff
@@ -173,63 +173,61 @@ func main() {
 
 ```csharp
 using Pulumi;
-using Pulumi.Kubernetes.Apps.V1;
 using Pulumi.Kubernetes.Types.Inputs.Core.V1;
 using Pulumi.Kubernetes.Types.Inputs.Apps.V1;
 using Pulumi.Kubernetes.Types.Inputs.Meta.V1;
+using System.Collections.Generic;
```

Comment:
Line 33:

>  `Program.cs` with a simple entry point, yo.

We should remove this block. Side note (unrelated to your changes): what the hell is this "yo"?

<img width="623" alt="image" src="https://user-images.githubusercontent.com/1454008/178218339-6906f363-b092-4604-a01b-cc30d501b7fc.png">

Reply:
I think I wanted to expand this "you define resources here". Fixes now